docs/installation-guide/quickstart.rst

```diff
@@ -173,39 +189,40 @@ twice to have 3 masters (bootstrap + 2 new master).
 
 .. code-block:: shell
 
-   $ kubectl apply -f new-master-node.yaml
-   node/new-master-node created
+   $ kubectl apply -f new-control-plane-node.yaml
```

Comment:
I would suggest that we systematically put --kubeconfig /etc/kubernetes/admin.conf for all kubectl commands because it is east to forget about KUBECONFIG variable when you switch from one console to another and when it is not set, error message is not obvious

Reply:
Agreed, @jbertran already suggested it some time ago.

Reply:
Agreed as well, got caught several time and the message is really unhelpful